api/src/coordinators/field.js

```
@@ -11,6 +21,32 @@ const addFieldToDocument = async function(documentId, body, file) {
   return BusinessField.createForDocument(documentId, fieldBody)
 }
 
+const addImageFieldFromUrlToDocument = async function(documentId, body, url) {
+  const filePath = path.resolve(__dirname, `../../uploads/${documentId}-${Date.now()}`)
+
+  const resp = await axios({ url, responseType: 'stream', method: 'get' })
+  const writeStream = fs.createWriteStream(filePath)
+  await promisifyStreamPipe(resp.data, writeStream)
+
+  const fileStats = await callFuncWithArgs(stat, filePath)
+
+  const urlParts = url.split('/')
+  const name = urlParts[urlParts.length - 1]
+
+  // This is mimic'ing multer's file object.
+  // Not ideal to be passing the multer object around, but that's a larger rewrite to fix
+  const file = {
+    originalname: name,
+    mimetype: lookup(url),
+    path: filePath,
+    size: fileStats.size
+  }
+
+  const result = await fieldCoordinator.addFieldToDocument(documentId, body, file)
+  fs.unlink(filePath, () => {}) // don't need to await
```

does this need to be wrapped in a try/catch to log/ignore error? otherwise we could get additional express errors along with it